src/main/java/org/jenkinsci/plugins/workflow/cps/CpsScmFlowDefinition.java

```diff
@@ -83,7 +86,7 @@ public boolean isLightweight() {
     @DataBoundSetter public void setLightweight(boolean lightweight) {
         this.lightweight = lightweight;
     }
-
+    
```

Comment:
BTW please configure your text editor to not make gratuitous whitespace changes to otherwise unmodified lines.